src/CellData/DiracDeltas.jl

```diff
@@ -112,7 +112,7 @@ function DiracDelta(model::DiscreteModel{D}, pvec::Vector{Point{D,T}}) where {D,
   cell_ids = collect(keys(cell_to_pindices))
```

Comment:
Do you want to attempt the usage of `Table` instead of a dictionary 
now that we have generalized `GenericQuadrature` ?

Reply:
Definitely Alberto, can you provide a link to the usage of `Table`? I ll attempt at adapting it here.

Reply:
There seem to be two libraries `Tables.jl` and `TypedTables.jl`.

Reply:
Correct. You can find some examples here. https://github.com/gridap/Gridap.jl/blob/master/test/ArraysTests/TablesTests.jl

Reply:
Great.. thank you, will through them as well.

Reply:
for the records ... we explored the solution based on `Table`. The code is here 

```julia
function DiracDelta(model::DiscreteModel{D}, pvec::Vector{Point{D,T}}) where {D,T}
  trian        = Triangulation(model)
  cache        = _point_to_cell_cache(KDTreeSearch(),trian)
  points2cell  = [_point_to_cell!(cache, pvec[i]) for i in 1:length(pvec)]
  cell_ids     = unique(points2cell)
  inv_cell_ids = Dict(cell_ids[i]=>i for i=1:length(cell_ids))

  data = Vector{eltype(pvec)}(undef,length(points2cell))
  ptrs = zeros(Int, length(cell_ids)+1)

  # Count sizes of each subarray
  for i=1:length(points2cell)
     j=inv_cell_ids[points2cell[i]]
     ptrs[j+1]+=1
  end
  length_to_ptrs!(ptrs)

  # Generate entries
  for (i, point) in enumerate(points2cell)
    j=inv_cell_ids[points2cell[i]]
    data[ptrs[j]]=pvec[i]
    ptrs[j]+=1
  end
  rewind_ptrs!(ptrs)

  pquad          = Vector{GenericQuadrature}(undef,length(cell_ids))
  weights_x_cell = [Fill(one(T),ptrs[i+1]-ptrs[i]) for i=1:length(cell_ids)]
  points_x_cell  = Table(data,ptrs)
  pquad = map(i -> GenericQuadrature(view(points_x_cell.data,ptrs[i]:ptrs[i+1]-1),
                                     weights_x_cell[i]), 1:length(cell_ids))

  trianv = view(trian,cell_ids)
  pmeas = Measure(CellQuadrature(pquad,
                                 points_x_cell,
                                 weights_x_cell,
                                 trianv,
                                 PhysicalDomain(),
                                 PhysicalDomain()))
  GenericDiracDelta{0,D,NotGridEntity}(trianv,pmeas)
end
```
HOWEVER, this solution seems to be slower and consume more memory! Not sure why, we will stick into the current solution given the evidence available.